src/Core/Domain/Alias/Command/UpdateAliasCommand.php

```diff
@@ -44,9 +51,35 @@ class UpdateAliasCommand
      */
     public function __construct(
         int $aliasId,
-        public readonly array $aliases,
-        public readonly string $searchTerm,
+        array $aliases,
```

Comment:
Are these VO objects or int values? Since the field and getters are not documented fully we can't guess
If the returned array is VOs, the input should still be an array of integers (transformed into AliasId in the constructor)